core/arch/aarch64/instr.c

```diff
@@ -136,6 +136,12 @@ instr_is_return(instr_t *instr)
     return (opc == OP_ret);
```

Comment:
> Unfortunately there are no Appendix A/B style documented Intel manuals available for AVX-512 in the same format as the existing x86 instruction sets.

Strange.  Do the AMD manuals have them in there?

> Add support to encoder for new VEX encodings. Fixed a bug to encode primary opcode byte in 3-byte VEX versions.

grammar consistency: s/Add/Adds/; s/Fixed/Fixes/.

> Adds raw encodings decoder test to x86 common.decoder test.

It's common.decode, not common.decoder.

Reply:
> Strange. Do the AMD manuals have them in there?

Don't think AMD even support AVX-512. No manuals yet.

> grammar consistency: s/Add/Adds/; s/Fixed/Fixes/.

Fixed.

> It's common.decode, not common.decoder.

Fixed.